### DISCUSSION TITLE
Number of schemas with a Postgres backend

### DISCUSSION THREAD

**Discussion by sumeetkh** [hasura/graphql-engine#9492] [Question]
We are using Hasura with Postgres database for a multi-tenant use case.

How many schemas can Hasura support on a single server. Is there some guideline or benchmarks around this?

Thank you!

**Comment by adas98012** [comment]
@sumeetkh 
Hasura can handle a large number of schemas, without any prescribed limit. That said, it's important to design your schema and database structure carefully to ensure optimal performance and scalability.

Here is an [article](https://hasura.io/blog/implementing-authorization-for-organization-based-team-based-or-tenant-based-apps-in-hasura/) that you may find useful. It is about implementing a multi-tenant app using Hasura.
